.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```

Comment:
Please fix the line ending with the new line. And check line ending style (should be Unix like)

Reply:
I would also extend gitignore a bit more.
Check please this example
https://github.com/guardian/frontend/blob/master/.gitignore